patches/api/0007-Adventure.patch

```diff
@@ -1352,10 +1352,18 @@ index 25a6f9313a1953def7470e411b53016f2ca14bef..d7a4cfed4f46b34f83fb2c07bdab5a71
 +    // Paper end
  }
 diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
-index edf4623a831442e1f06daabc402a3f32610dd519..8ed69c4be84db742f2ff53adfd40f9eb8823e02d 100644
+index 6c910e09eda6f4f08226ccc75189171015a72b85..fd5e85e2b20a88ab245eba113fe67f2ac18ac691 100644
 --- a/src/main/java/org/bukkit/entity/Player.java
 +++ b/src/main/java/org/bukkit/entity/Player.java
-@@ -33,7 +33,28 @@ import org.jetbrains.annotations.Nullable;
+@@ -1,6 +1,7 @@
+ package org.bukkit.entity;
+ 
+ import java.net.InetSocketAddress;
++import java.util.List; // Paper
```

Comment:
Looks like you left this in

Reply:
That’d be because of my Javadoc settings in IntelliJ to import automatically… I’ll fix tomorrow if you don’t handle it first